rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -324,3 +333,82 @@ def get(self,
                 substr_filters=search
             )
         return inter_deployment_dependencies
+
+
+class InterDeploymentDependenciesRestore(SecuredResource):
+    @authorize('inter_deployment_dependency_create')
+    def post(self):
+        """
```

Comment:
PEP 257 is the style document for docstrings (like PEP8 for code), and it prefers the style:
```python
"""Short summary, just one line

Longer text longer text
probably even multiline

:param: maybe some param declarations here
"""
```

Reply:
Nice, made me think also about the parameter 'runtime_only_evaluation' and I realized there is no need in it so I deleted it.